src/runner/main.cpp

```diff
@@ -251,6 +251,10 @@ toast_notification_handler_result toast_notification_handler(const std::wstring_
 
         return toast_notification_handler_result::exit_success;
     }
+    else if (param == L"update_abort/")
+    {
+        return toast_notification_handler_result::exit_success;

```

Comment:
why did we made "abort update" an activatable button if we don't do anything with it? let's make it a simple button, if we don't plan to do any actions for it, or perhaps we need to remove the downloaded installer here.

Reply:
Where do we have the "abort update" button? I haven't seen it in the screenshots.

Reply:
@htcfreek, when update is ready to install. There are 3 options: 'Update now', 'At next launch', 'Abort'.